text/0000-compact-slates.md

```diff
@@ -0,0 +1,240 @@
+        }
+      ],
+      "kernels": [
+        {
+          "features": "Plain",
+          "fee": "7000000",
+          "lock_height": "0",
+          "excess": "000000000000000000000000000000000000000000000000000000000000000000",
+          "excess_sig": "00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+        }
+      ]
+    }
+  },
+  "amount": "60000000000",
+  "fee": "7000000",
+  "height": "5",
+  "lock_height": "0",
+  "ttl_cutoff_height": null,
+  "payment_proof": null,
+  "participant_data": [
+    {
+      "id": "0",
+      "public_blind_excess": "033ac2158fa0077f087de60c19d8e431753baa5b63b6e1477f05a2a6e7190d4592",
+      "public_nonce": "031b84c5567b126440995d3ed5aaba0565d71e1834604819ff9c17f5e9d5dd078f",
+      "part_sig": null,
+      "message": null,
+      "message_sig": null
+    }
+  ]
+}
+```
+
+In version 4 of the Slate format, the initial Slate now looks like the following:
+
+```
+{
+  "ver": "4.3",
+  "id": "mavTAjNm4NVztDwh4gdSrQ",
+  "amt": "1000000000",
+  "fee": "8000000",
+  "hgt": "437088",
+  "sigs": [
+    {
+      "excess": "02ef37e5552a112f829e292bb031b484aaba53836cd6415aacbdb8d3d2b0c4bb9a",
+      "nonce": "03c78da42179b80bd123f5a39f5f04e2a918679da09cad5558ebbe20953a82883e"
+    }
+  ]
+}
+```
+
+The 'return' slate from the recipient to the originator is expected to be larger, as it must include the recipient's outputs/inputs and proofs (as the party who posts the transaction needs all of this information). However, compacting the first-leg journey allows for future possibilities that may have been more difficult in previous iterations of the slate format. For instance, an initiator taking Grin payments might be expected to be better equipped to receive Grin transactions. They could present the payer with an invoice transaction for the amount compacted to the size of a QR code, which the payer could scan, accept, and then post back to the invoicer's listening wallet infrastructure.
+
+Compacting the slate also acts as a minor privacy-enhancer by hiding the initiator's outputs from the other party.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+### Slate Definition
+
+Entries prefixed with `//` denote fields that may be omitted, as well as their default assumed values
+
+```
+{
+  "ver": "4.3",
+//"num_parts: 2,
+  "id": "mavTAjNm4NVztDwh4gdSrQ",
+//"tx": null,
+  "amt": "1000000000",
+  "fee": "8000000",
+  "hgt": "437088",
```

Comment:
Did you figure out why this was needed? I don't believe it is, or at minimum, it can probably be optional

Reply:
I don't think it's needed at all, so will try removing it.